packages/commercetools/theme/components/AppHeader.vue

```diff
@@ -22,6 +22,7 @@
           <SfButton
             v-e2e="'app-header-account'"
             class="sf-button--pure sf-header__action"
+            aria-label="open-account-button"
```

Comment:
Shouldn't `aria-label` value be written user friendly? For example: "Open account button".

Reply:
Yes, you are right. I will change it

Reply:
Thanks!